docs/developing/integrate/neon_transfer.mdx

```diff
@@ -0,0 +1,182 @@
+console.log(solanaSignature);
+```
+
+</TabItem>
+<TabItem value="Opt2" label="Neon EVM to Solana" default>
+
+Similarly, to transfer NEON from Neon EVM to Solana, use this pattern:
+
+```javascript
+const transaction = await neonNeonWeb3Transaction(web3, neonWallet, NEON_TRANSFER_CONTRACT_DEVNET, solanaWallet, amount); // Neon EVM Transaction object
+const neonSignature = await sendNeonTransaction(web3, transaction, neonWallet); // method for sign and send transaction to network
+console.log(neonSignature);
+```
+
+</TabItem>
+</Tabs>
+
+
+### Transfer ERC-20 tokens
+
+When working with Devnet, Testnet, or Mainnet, different ERC-20 tokens are utilized. We maintain a JSON [list of supported tokens](https://github.com/neonlabsorg/token-list). Alternatively, please refer to the [token list](/docs/tokens/token_list.md).
+
+<!-- todo we need a single source of truth not 2 different pages to go to which may drift! Suggest JSON as API to feed the docs site -->
+
+<Tabs>
+  <TabItem value="Opt1" label="Solana to Neon EVM" default>
+
+To transfer ERC-20 tokens from Solana to Neon EVM, use the following patterns:
+
+```javascript
+const token = tokenList[0];
+const transaction = await neonTransferMintWeb3Transaction(connection, web3, proxyApi, proxyStatus, neonEvmProgram, solanaWallet, neonWallet, token, amount);
+transaction.recentBlockhash = (await connection.getLatestBlockhash()).blockhash;
+const solanaSignature = await sendSolanaTransaction(connection, transaction, [signer], true, { skipPreflight: false });
+console.log(solanaSignature);
+```
+
+</TabItem>
+<TabItem value="Opt2" label="Neon EVM to Solana" default>
+
+Similarly, to transfer ERC-20 tokens from Neon EVM to Solana:
+
+```javascript
+const token = tokenList[0];
+const mintPubkey = new PublicKey(token.address_spl);
+const associatedToken = getAssociatedTokenAddressSync(mintPubkey, solanaWallet);
+const solanaTransaction = createMintSolanaTransaction(solanaWallet, mintPubkey, associatedToken, proxyStatus);
+solanaTransaction.recentBlockhash = (await connection.getLatestBlockhash()).blockhash;
+const neonTransaction = await createMintNeonWeb3Transaction(web3, neonWallet.address, associatedToken, token, amount);
+const signedSolanaTransaction = await sendSolanaTransaction(connection, solanaTransaction, [signer], true, { skipPreflight: false });
+const signedNeonTransaction = await sendNeonTransaction(web3, neonTransaction, neonWallet);
+console.log(signedSolanaTransaction, signedNeonTransaction);
+```
+
+</TabItem>
+</Tabs>
+
+
+## Alternative libraries
+
+Within the Neon Transfer codebase, we employ the [web3.js](https://web3js.readthedocs.io/en/v1.10.0/) library to streamline our code. However, you can opt for alternatives such as [@solana/web3.js](https://docs.solana.com/developing/clients/javascript-api), [ethers.js](https://docs.ethers.org/v6/), or [WalletConnect](https://walletconnect.com/).
```

Comment:
I expanded the list to include Solana web3 js library, as per the original prerequisites list. Pls confirm @oable

Reply:
No @solana/web3 is library only for the Solana network. Web3.js, ethers.js, WalletConnect for the Ethereum like Neon EVM.